crypto/sha/asm/keccak1600-ppc64.pl

```diff
@@ -668,6 +668,8 @@
 	subi	$out,r4,1		; prepare for stbu
 	mr	$len,r5
 	mr	$bsz,r6
+	${UCMP}i r7,1                   ; r7 = 'next' argument
+	blt	.Lnext_block
```

Comment:
I don't know ppc, but this seems to be inverted from the others. It's branching to next_block if next < 1, but the others go to next_block if next != 0.

Reply:
Thanks @davidben. Yes the following will work:
```
${UCMP}i $r7,0
bne      .Lnext_block
```